eng/pipelines/common/upload-unsigned-artifacts-step.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
+  name: ''
+
+steps:
+- task: CopyFiles@2
```

Comment:
Will merging Shipping and NonShipping assets, cause any troubles to loose track which assets are actually Shipping and NonShipping when publishing to BAR?

Reply:
> merging Shipping and NonShipping assets

`flattenFolders` defaults to `false`, so this doesn't happen. Would certainly be a problem!

(The rest of the build relies heavily on the implicit `false` too, so I don't think it's worth adding a code comment about it here in particular.)

Reply:
Ah I missed the `flattenFolders` parameter existed and that it defaulted to false. Cool, thanks for clarifying.